gw2api/objects/base_object.py

```diff
@@ -94,7 +94,10 @@ def get(self, url=None, **kwargs):
         request_url = request_url.strip('&')  # Remove any trailing '&'
         request_url = request_url.strip('?')  # Remove any trailing '&'
         print(request_url)
```

Comment:
```suggestion
```
Not exactly part of the PR but we shouldn't have print statement lying around. If we need that I'm happy to introduce logging in the future.